store/etcd/v2/etcd.go

```diff
@@ -99,7 +98,7 @@ func New(addrs []string, options *store.Config) (store.Store, error) {
 	if options != nil && options.SyncPeriod != 0 {
 		go func() {
 			for {
-				c.AutoSync(context.Background(), options.SyncPeriod)
+				_ = c.AutoSync(context.Background(), options.SyncPeriod)
```

Comment:
why assign this value to nothing? Is this more idiomatic?

Reply:
if you want to ignore an error, it's better to do it explicitly.